Depend-on/Bash.sh

```diff
@@ -0,0 +1,221 @@
+                    echo "ffmpeg"
+                    echo "libavcodec-dev libavformat-dev libavutil-dev libswscale-dev"
+                    echo "freerdp2-dev"
+                    echo "libpango1.0-dev"
+                    echo "libssh2-1-dev"
+                    echo "libtelnet-dev"
+                    echo "libvncserver-dev"
+                    echo "libwebsockets-dev"
+                    echo "libpulse-dev"
+                    echo "libssl-dev"
+                    echo "libvorbis-dev"
+                    echo "libwebp-dev"
+
+                    read -p "Do you understand? (yes/no): " choice
+                    case "$choice" in 
+                        yes|YES|y|Y)
+                            sudo apt-get install -y ffmpeg
+                            sudo apt-get install -y libavcodec-dev libavformat-dev libavutil-dev libswscale-dev
+                            sudo apt-get install -y freerdp2-dev
+                            sudo apt-get install -y libpango1.0-dev
+                            sudo apt-get install -y libssh2-1-dev
+                            sudo apt-get install -y libtelnet-dev
+                            sudo apt-get install -y libvncserver-dev
+                            sudo apt-get install -y libwebsockets-dev
+                            sudo apt-get install -y libpulse-dev
+                            sudo apt-get install -y libssl-dev
+                            sudo apt-get install -y libvorbis-dev
+                            sudo apt-get install -y libwebp-dev
+                            ;;
+                        *)
+                            echo "Installation cancelled."
+                            ;;
+                    esac
+                elif command -v dpkg &>/dev/null; then
+                    echo "The following packages will be installed:"
+                    echo "ffmpeg"
+                    echo "libavcodec-dev libavformat-dev libavutil-dev libswscale-dev"
+                    echo "freerdp2-dev"
+                    echo "libpango1.0-dev"
+                    echo "libssh2-1-dev"
+                    echo "libtelnet-dev"
+                    echo "libvncserver-dev"
+                    echo "libwebsockets-dev"
+                    echo "libpulse-dev"
+                    echo "libssl-dev"
+                    echo "libvorbis-dev"
+                    echo "libwebp-dev"
+
+                    read -p "Do you understand? (yes/no): " choice
+                    case "$choice" in 
+                        yes|YES|y|Y)
+                            sudo dpkg -i <package.deb>
+                            ;;
+                        *)
+                            echo "Installation cancelled."
+                            ;;
+                    esac
+                else
+                    echo "Neither apt-get nor dpkg found. Unsupported package manager."
+                fi
+                ;;
+            *)
```

Comment:
I don't think making the assumption that anything that isn't Ubuntu or EL is automatically SuSE is the right thing to do. While EL-based, Debian-based, and SuSE-based covers most available distributions, there are others out there (like Alpine, which is what we use for the Docker image) and other ways of managing packages.

If you/we don't want to build out and maintain package commands for each of these, that's fine, but then the `*)` should throw an error, not assume that everything not EL or Ubuntu-based is SuSE.

Reply:
any thing simpler for my 12yo brain to understand?


what I believe your talking about is the pkg managers and not working if it errors all of the time 

see the redundancy talk of my idea

Reply:
My preference would be to create a "SuSE" section of the `case` statement above this `*` (default) handler, and put the SuSE commands in that section, and then use the `*` area to provide feedback that the distribution is either undetected or unsupported. Roughly, something like this:
```
case "distro" in
    CentOS|RedHatEnterpriseServer|Rocky)
        // Run EL commands here
    Debian|Ubuntu|Xubuntu)
        // Run Debian commands here
    SuSE)
        // Run SuSE commands here
    *)
        // Throw error output and exit - distro not detected, not supported, etc.
esac
```